src/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  env: {
+    browser: true,
+    node: false,
+  },
+  extends: [
+    "plugin:vue/vue3-essential",
+    "eslint:recommended",
+    "@vue/eslint-config-typescript/recommended",
+    "@vue/eslint-config-prettier",
+  ],
+  parserOptions: {
+    parser: "@typescript-eslint/parser",
+    ecmaVersion: 12,
+    sourceType: "module",
+  },
+};
```

Comment:
This is the second eslint file, specifically made for all those frontend Vue scripts